src/core/crypto/ecdsa.hpp

```diff
@@ -204,6 +217,102 @@ class P256
         }
     };
 
+#if OPENTHREAD_CONFIG_PLATFORM_KEY_REFERENCES_ENABLE
+    /**
+     * This class represents a key pair (public and private keys) as a PSA KeyRef.
+     *
+     * The key pair is stored using Distinguished Encoding Rules (DER) format (per RFC 5915).
```

Comment:
Remove this line? Not relevant/correct anymore?

Reply:
Will push a fix in next commit.